### PR TITLE
feat: remove `@loadable/component` in streaming ssr

### PR DESCRIPTION
**Pull request by xuchaobei**
# PR Details

## Description

Remove `@loadable/component` in streaming ssr.

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [x] I have updated changeset
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/runtime/plugin-runtime/src/ssr/serverRender/renderToStream/template.ts [hunk 1]
```diff
@@ -1,74 +1,20 @@
...
-  const {
-    jsx,
-    chunkExtractor,
-    styleSheet,
-  }: {
-    context?: RuntimeContext;
-    jsx: React.ReactElement;
-    chunkExtractor?: ChunkExtractor;
-    styleSheet?: ServerStyleSheet;
-  } = [getStyledComponentCss, getLoadableChunks].reduce(
```
Comment (GiveMe-A-Name) [inline packages/runtime/plugin-runtime/src/ssr/serverRender/renderToStream/template.ts:34]: styledComponent 也不在 Streaming SSR 中支持了吗

Comment (xuchaobei) [reply packages/runtime/plugin-runtime/src/ssr/serverRender/renderToStream/template.ts:34]: css in js 在 streaming ssr 里现在是支持不了的，需要库本身做改造

Comment (GiveMe-A-Name) [reply packages/runtime/plugin-runtime/src/ssr/serverRender/renderToStream/template.ts:34]: 具体是有啥问题呀？
之前跑 Example 的时候 css in js 在 streaming SSR 表现正常

Comment (xuchaobei) [reply packages/runtime/plugin-runtime/src/ssr/serverRender/renderToStream/template.ts:34]: https://github.com/styled-components/styled-components/issues/3658

**Review by GiveMe-A-Name — APPROVED** [review]

**Pull request merged**